cfgov/unprocessed/apps/paying-for-college/js/views/school-view.js

```diff
@@ -56,6 +85,18 @@ const schoolView = {
     }, 500 );
   },
 
+  _handleProgramSelectChange: function( event ) {
+    const salary = event.target.options[event.target.selectedIndex].dataset.programSalary;
```

Comment:
Maybe pull `event.target` into its own variable so you don't need to look it up a bunch in this function.

```
const target = event.target;
const salary = target.options[target.selectedIndex].dataset.programSalary;
const programName = target.options[target.selectedIndex].innerText;
let pid = target.value;
…
```

Reply:
Good catch! I obviously failed to refactor after my first pass. 😄